### PR TITLE
add event payload examples

### PR DESCRIPTION
**Pull request by jchen293**
add event payload code examples

### CODE REVIEW

File: official/docs/curl/current/events/retrieve-all-payloads.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+curl -X GET https://api.easypost.com/v2/events/:event_id/payloads \
```
Comment (Justintime50) [inline official/docs/curl/current/events/retrieve-all-payloads.sh:1]: Please use the proper ID conventions we've used elsewhere by specifying the prefix underscore dots pattern

File: official/docs/curl/current/events/retrieve-payload.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+curl -X GET https://api.easypost.com/v2/events/:event_id/payloads/:id \
```
Comment (Justintime50) [inline official/docs/curl/current/events/retrieve-payload.sh:1]: Please use the proper ID conventions we've used elsewhere by specifying the prefix underscore dots pattern

File: official/docs/curl/current/events/retrieve-all-payloads.sh (hunk 1 above)
Comment (jchen293) [reply official/docs/curl/current/events/retrieve-all-payloads.sh]: resolved

File: official/docs/curl/current/events/retrieve-payload.sh (hunk 2 above)
Comment (jchen293) [reply official/docs/curl/current/events/retrieve-payload.sh]: resolved

**Review by Justintime50 — APPROVED** [review]

File: official/docs/csharp/current/events/retrieve-all-payloads.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+{
+    public class Examples
+    {
+        public static async Task Main()
+        {
+            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;
+
+            var client = new EasyPost.Client(apiKey);
+
+            List<Payload> payloads = await Client.Event.RetrieveAllPayloads("evt_...");
```
Comment (nwithan8) [inline official/docs/csharp/current/events/retrieve-all-payloads.cs:18]:
```suggestion
            List<Payload> payloads = await client.Event.RetrieveAllPayloads("evt_...");
```

File: official/docs/csharp/current/events/retrieve-payload.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+{
+    public class Examples
+    {
+        public static async Task Main()
+        {
+            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;
+
+            var client = new EasyPost.Client(apiKey);
+
+            Payload payload = await Client.Event.RetrievePayload("evt_...", "payload_...");
```
Comment (nwithan8) [inline official/docs/csharp/current/events/retrieve-payload.cs:18]:
```suggestion
            Payload payload = await client.Event.RetrievePayload("evt_...", "payload_...");
```

**Review by nwithan8 — CHANGES_REQUESTED** [review]

**Review by nwithan8 — APPROVED** [review]

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**